src/api/api.cc

```diff
@@ -5191,11 +5191,26 @@ MaybeLocal<v8::Context> v8::Object::GetCreationContext() {
   return MaybeLocal<v8::Context>();
 }
 
+extern "C" void V8RecordReplayGetDefaultContext(v8::Isolate* isolate, v8::Local<v8::Context>* cx);
+
 Local<v8::Context> v8::Object::GetCreationContextChecked() {
   Local<Context> context;
-  Utils::ApiCheck(GetCreationContext().ToLocal(&context),
-                  "v8::Object::GetCreationContextChecked",
-                  "No creation context available");
+  if (!GetCreationContext().ToLocal(&context)) {
```

Comment:
Let's remove this change. I suggested an alternative [here](https://linear.app/replay/issue/TT-957/recorder-crash-in-getcreationcontextchecked#comment-f2a8d32a).